packages/zosjobs/src/SubmitJobs.ts

```diff
@@ -248,14 +247,30 @@ export class SubmitJobs {
             });
             return activeJob;
         }
-        // if viewAppSpoolContent option passed, it waits till job status is output
-        // then get content of each spool file and return array of ISpoolFiles object
-        if (parms.viewAllSpoolContent || parms.waitForOutput) {
+        // wait for job status to be OUTPUT for all other situations
+        if (parms.task != null) {
+            parms.task.statusMessage = "Waiting for " + responseJobInfo.jobid + " to enter OUTPUT";
+            parms.task.percentComplete = TaskProgress.THIRTY_PERCENT;
+        }
+        const job: IJob = await MonitorJobs.waitForJobOutputStatus(session, responseJobInfo);
```

Comment:
This logic needs to stay inside the "if" statements below, so that it only runs when `--directory` or `--wait-for-output` are specified. Moving it out will cause the `Waiting for <jobid> to enter OUTPUT` message to appear on all "submit jobs" commands.